ClientApp/src/app/news-feed/news-feed.component.scss

```diff
@@ -0,0 +1,16 @@
+.news-thumbnail {
+  width: 220px;
+  height: 123px;
+}
```

Comment:
Codacy found an issue: [Expected newline after "{" of a multi-line block (block-opening-brace-newline-after)](https://app.codacy.com/gh/InvestingOak/InvestingOak/pullRequest?prid=6318563)